app/main.py

```diff
@@ -1,8 +1,28 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    for person in people:
+        name = person["name"]
+        age = person["age"]
+
+        person_instance = Person(name, age)
+
+        Person.people[name] = person_instance
```

Comment:
This line is redundant because the `Person` instance is already added to `Person.people` in the constructor. You can remove this line to avoid unnecessary reassignment.